src/ToggleIcon/VaToggleIcon.vue

```diff
@@ -58,24 +59,40 @@
     },
     data () {
       return {
-        actived: this.active,
+        isActive: this.active,
         style: {
           display: 'inline-block',
-          width: this.size,
-          height: this.size
+          width: this.$attrs.size,
+          height: this.$attrs.size,
+          position: 'relative'
         }
       }
     },
     methods: {
       toggle () {
-        this.actived = !this.actived
+        this.isActive = !this.isActive
       },
-      show () {
-        this.actived = true
+      onActivate () {
+        this.isActive = true
       },
-      hide () {
-        this.actived = false
+      onDeactivate () {
+        this.isActive = false
       }
     }
   }
 </script>
+
+<style>
```

Comment:
not scoped, but also maybe not needed as a class since it only defines one attribute

Reply:
It's scoped already. In recent commits :)